timur/lib/client/jsx/components/model_map/model_node.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 
-const ModelNode = ({model_name, center, size, selected, handler, disabled}) =>
+const ModelNode = ({model_name, center, size, selected, handler, disabled, numSiblings}) =>
+
   center ? (
     <div
       className={`model_node ${selected ? 'selected' : ''} ${
         disabled ? 'disabled' : ''
       }`}
-      style={{top: center.y, left: center.x}}
+      style={{top: center.y, left: center.x, '--model-width': `${600 / (numSiblings + 1) - 15}px` }}
```

Comment:
This is a pretty cool trick (paired with `var(--model-width)` in the css) that lets you pass data into any css styles set on this thing without having to override the style itself (so precedence isn't destroyed).

Reply:
oh neat. Is this a React thing, where you can pass in / use CSS variables?

Reply:
pretty cool!

Reply:
Apparently, pure CSS/html.